docs/advanced/naming.md

```diff
@@ -1,190 +1,262 @@
-    - But where is this from? BluRay? DVD? WEB-DL? WEBRip? Oh web-dl? Which one?
-    - Is it an encode or a remux?
-    - What audio/video codec is in this? My TV can only play XYZ.
-
-    Example 2: `[GJM] Mobile Suit Gundam - The Witch from Mercury - 24 [AEB8BE46].mkv`
-    - First look tells me it's a show called Mobile Suit Gundam - The Witch from Mercury by GJM.
-    - What resolution is this? I only want 1080p!
-    - But where is this from? BluRay? DVD? WEB-DL? WEBRip? This upload is after the BluRay released so maybe it's BluRay. (WRONG, it's WEB)
-    - Is it an encode or a remux?
-    - What audio/video codec is in this? My TV can only play XYZ.
-
-    Example 3: `[SubsPlease] Boku no Hero Academia - 90 (1080p) [4B8B1261].mkv`
-    - First look tells me it's a show called Boku no Hero Academia by SubsPlease.
-    - This show has 6 seasons which one is this?
-    - But where is this from? BluRay? DVD? WEB-DL? WEBRip? Oh web-dl? Which one?
-    - Is it an encode or a remux?
-    - What audio/video codec is in this? My TV can only play XYZ.
-
-    As a thought excercise, here's what it looks like to both humans and machines: `[SomeGroup] Some Show - 17 [CRC].mkv`
-    
-    The example is a real release which I've simply replaced the names to simulate how it looks to someone who's not acquainted with the whole anime scene. Now tell me:
-
-    - What source is it?
-    - What resolution is it?
-    - What codecs are in it?
-    - Is this season 1 with 24 episodes or is it some other season with absolute numbering?
-    - Is it an encode or a remux?
-
-    Now let's try the same thought excercise on something that's actually named properly:
-    
-    ```
-    Some.Show.S03E04.1080p.CR.WEB-DL.Dual-Audio.AAC2.0.H.264-SomeGroup.mkv
-    [SomeGroup] Some Show - S03E04 (BD 1080p HEVC Opus) [Dual Audio] [CF1029D9].mkv
-    Some.Show.S03E04.1080p.BluRay.Remux.Dual-Audio.FLAC2.0.H.264-SomeGroup.mkv
-    ```
-    Here, from just one look, I know exactly:
-    - What show they are, that is, `Some Show`
-    - Which episode of which season they are, that is, `S03E04`
-    - What sources they are. First is a Crunchyroll WEB-DL, second is a BD encode, and the third one is a remux.
-    - What resolution they are, that is, 1080p and that's what I want.
-    - What codecs have been used, so I can judge it's compatibility with my device before grabbing it.
-
-2. You lose downloads from automatic snatchers because they simply fail to parse stupidly named releases.
-3. People usually have to rename said stupidly named releases to make them work in their media servers, which means they can no longer seed it.
-4. Switching to a more compatible naming scheme has no disadvantages. It won't harm you or the quality of your release, it'll only serve to widen your audience and improve convenience. It should be a no-brainer considering that it only benefits those who are interested in the release.
```

Comment:
Can you incorporate this in some way? The job of this section was to *sell* this page to releasers who are dead set on their ways. A guide won't convince them that easily which is why this section was the "marketing" part